core/components/com_members/admin/controllers/members.php

```diff
@@ -159,8 +165,18 @@ public function displayTask()
 				$note
 					->select('id')
 					->select('user_id');
+			}])
+			->including(['auths', function ($auth) {
+				$auth
+					->select('*');
 			}]);
 
+		if ($filters['status'])
```

Comment:
I was trying to filter the user log records by its status. Still working on that